runtime/src/lib.rs

```diff
@@ -689,6 +693,7 @@ impl pallet_recovery::Config for Runtime {
     type FriendDepositFactor = FriendDepositFactor;
     type MaxFriends = MaxFriends;
     type RecoveryDeposit = RecoveryDeposit;
+    type WeightInfo = ();
```

Comment:
Can't we use something like this `pallet_recovery::weights::SubstrateWeight<Runtime>;` instead of `()`

Reply:
We can indeed. Done